pkger/parser.go

```diff
@@ -418,11 +409,6 @@ func (p *Pkg) addObjectForRemoval(k Kind, pkgName string, id influxdb.ID) {
 	}
 
 	switch k {
-	case KindBucket:
```

Comment:
these are all moving to the stateCoordinator, which tracks the new/exists/to remove state of a resource